src/Rubric/RubSmalltalkEditor.class.st

```diff
@@ -818,6 +818,12 @@ RubSmalltalkEditor >> isSmalltalkEditor [
 	^ true 
 ]
 
+{ #category : #private }
+RubSmalltalkEditor >> isWordCharacter: aCharacter [
+	"In smalltalk code, colons make part of selectors too and are part of a word"
+	^ aCharacter isAlphaNumeric or: [aCharacter = $:]
```

Comment:
I am able to get into trouble on this if I try to edit an assignment where there is no space between the variable and the `:=`. 
<img width="111" alt="image" src="https://user-images.githubusercontent.com/17723745/143469629-3c16e306-f252-41dd-b1da-3ff017027a6e.png">
when I start typing `dumb` I get a completion for `dumb`, accepting the completion removes the `:`

Reply:
Very good catch! Thanks Kasper, I'll make a test on that.